src/screens/Users/index.tsx

```diff
@@ -109,16 +112,17 @@ const UserListScreen: VFC = () => {
   );
 
   const errorText = getErrorMessage(error);
-  const data = Object.values(users).sort(sortByLastCreated);
+  const data = users.sort(sortByLastCreated);
```

Comment:
I believe you can memoize this to reduce re-renders.

```suggestion
  const data = useMemo(() => users.sort(sortByLastCreated), [users, sortByLastCreated]);
```